web/modules/custom/dpl_reservations/src/Form/ReservationListSettingsForm.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\dpl_reservations\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Reservation list setting form.
+ */
+class ReservationListSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames(): array {
+    return [
+      'reservation_list.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'reservation_list_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('reservation_list.settings');
+
+    $form['settings'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Basic settings'),
+      '#tree' => FALSE,
+    ];
+
+    $form['settings']['pause_reservation_info_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Pause reservation link'),
+      '#description' => $this->t('The link in the pause reservation modal'),
```

Comment:
Please remember to add a context for all the translations in this file.

Reply:
Fixed